### PR TITLE
Add effect to load overdue search history

### PR DESCRIPTION
**Pull request by msasikanth**
https://app.shortcut.com/simpledotorg/story/8520/display-search-history-in-overdue-search-screen

### CODE REVIEW

**Comment by msasikanth** [comment]
Current dependencies on/for this PR:
* master
  * **PR #3951** <a href="https://app.graphite.dev/github/pr/simpledotorg/simple-android/3951" target="_blank">Graphite</a> 
    * **PR #3952** <a href="https://app.graphite.dev/github/pr/simpledotorg/simple-android/3952" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/simpledotorg/simple-android/3952?utm_source=stack-comment).

File: app/src/main/java/org/simple/clinic/home/overdue/search/OverdueSearchModel.kt [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package org.simple.clinic.home.overdue.search
+
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+data class OverdueSearchModel(
+    val overdueSearchHistory: Set<String>?
```
Comment (janhavisinghh) [inline app/src/main/java/org/simple/clinic/home/overdue/search/OverdueSearchModel.kt:8]: Why did you go for a `Set` instead of a `String`?

Comment (janhavisinghh) [reply app/src/main/java/org/simple/clinic/home/overdue/search/OverdueSearchModel.kt:8]: List*

**Review by janhavisinghh — APPROVED** [review]

**Review by janhavisinghh — APPROVED** [review]

**Comment by msasikanth** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/simpledotorg/simple-android/3952?utm_source=stack-comment).


Job Reference: QO6BNrFvMbihWnEksjTV

**Pull request merged**